setup.py

```diff
@@ -30,14 +30,14 @@
     zip_safe=False,
     install_requires=[
         'click',
-        'docker-py',
+        'docker',
         'pip',
         'PyYAML',
         'retrying',
         'requests',
-        'scrapinghub>=2.0.3',
+        'scrapinghub>=2.3.1',
         'six>=1.7.0',
-        'tqdm',
+        'tqdm==4.11.2',
```

Comment:
Same here, any breaking changes or just an accidental update?

Reply:
Hey @vshlapakov , yes there is a reason. we need `tqdm==4.11.2` because newer versions generates a little bit different progress bars and we have test cases strictly defined for this progress bars, tests are failing with newer `tqdm`. 

Example of how our test cases looks like bellow, check how dependent of the current bar format we are, I'd say that to update `tqdm` it'd be ideal to do in a separated PR but I don't think it is something we'd like to do now as it is working:

![image](https://user-images.githubusercontent.com/163367/85519272-9cb62400-b5d7-11ea-8701-fd3a30f7b0d6.png)

![image](https://user-images.githubusercontent.com/163367/85519309-acce0380-b5d7-11ea-86c3-72e4770f723d.png)

![image](https://user-images.githubusercontent.com/163367/85519391-cff8b300-b5d7-11ea-8aa6-84be740dc77c.png)

Reply:
@vshlapakov We didn't notice it before because tox is using the pinned libraries from the `requirements.txt` to run. It means that, if something change in, lets say, `click` we won't notice it until a user report it to us as our test cases aren't using unpinned libs.

Reply:
Oh, good point, let's address it separately then, thanks for explanation :+1: